pkg/moov/bank_account_models.go

```diff
@@ -135,3 +140,17 @@ const (
 	AchReturnCode_R38 AchReturnCode = "R38"
 	AchReturnCode_R39 AchReturnCode = "R39"
 )
+
+// RTPRejectionCode is a rejection code of an RTP transaction that caused the bank account status to change.
+type RTPRejectionCode string
+
+// List of RTPRejectionCode
+const (
+	RTPRejectionCode_AC03 RTPRejectionCode = "AC03" // Account Invalid
+	RTPRejectionCode_AC04 RTPRejectionCode = "AC04" // Account Closed
+	RTPRejectionCode_AC06 RTPRejectionCode = "AC06" // Account Blocked
+	RTPRejectionCode_AC14 RTPRejectionCode = "AC14" // Creditor Account Type Invalid
+	RTPRejectionCode_AG01 RTPRejectionCode = "AG01" // Transactions Forbidden On Account
+	RTPRejectionCode_AG03 RTPRejectionCode = "AG03" // Transaction Type Not Supported
+	RTPRejectionCode_MD07 RTPRejectionCode = "MD07" // Customer Deceased
+)
```

Comment:
In the [docs](https://docs.moov.io/api/sources/bank-accounts/list/), I noticed these enum values have descriptions in them as well: `AC03 - Account Invalid`; do we need to include that here?

Reply:
Good call out. I think we want to return AC03 instead of `AC03 - Account Invalid`. The `description` field has the code and text.